src/LAMMPS.jl

```diff
@@ -224,13 +224,20 @@ end
 ```
 """
 function command(lmp::LMP, cmd::Union{String, Array{String}})
-    if cmd isa String
-        API.lammps_commands_string(lmp, cmd)
-    else
-        API.lammps_commands_list(lmp, length(cmd), cmd)
-    end
+    try
+        if cmd isa String
+            task = @async API.lammps_commands_string(lmp, cmd)
```

Comment:
Why do we need a task? Can't we just catch the InterruptException?

Reply:
I think I'll have to look into this in more detail. For my toy-example, interupting the lammps instance seems to actually work without any modifications to the code. However, when I try to interupt my real simulation, it doesn't work.

I don't know why that would be the case.